src/Methods/Modular_Programming/Hard/Q1/RightPrimeTest.java

```diff
@@ -0,0 +1,45 @@
+package Methods.Modular_Programming.Hard.Q1;
+import global.BaseTest;
+import global.variables.*;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.RepetitionInfo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RightPrimeTest extends BaseTest{
+
+    public Clause[] testSentence() {
+        provideInput("5");
```

Comment:
initialize input outside as int/String input = 5 and then put it into provide input.

Reply:
Why? I don't use it anywhere else I don't think.

Reply:
I'd agree with Quan here, it's not used elsewhere but eventually you'll need to be providing an array of inputs instead of just one so the structure will look more like having a class variable defined at the top

Reply:
Sorry, can you reword that, I don't think I'm quite understanding what you mean. When you say I'll eventually need to provide an array of inputs, do you mean in future tests? or later in this test? It doesn't have the same format as my later inputs now that they're parameterized, and I don't see the point of adding unnecessary code

Reply:
To reword, I think defining a `String input = "5";` outside of the testSentence() method is the move as it will be easier to switch to the `InputTest` framework Abhi and I make later

Reply:
okay, thanks for the clairifcation